.craft.yml

```diff
@@ -2,20 +2,16 @@ minVersion: '0.23.1'
 changelogPolicy: simple
 preReleaseCommand: bash scripts/craft-pre-release.sh
 targets:
-  #
-  # Deactivated for now. This needs to be reactivated if the new Sentry Lambda Extension is deployed to production.
-  # (ask Anton Pirker if you have questions.)
-  #
-  # - name: aws-lambda-layer
-  #   includeNames: /^sentry-node-serverless-\d+.\d+.\d+(-(beta|alpha)\.\d+)?\.zip$/
-  #   layerName: SentryNodeServerlessSDK
-  #   compatibleRuntimes:
-  #     - name: node
-  #       versions:
-  #         - nodejs10.x
-  #         - nodejs12.x
-  #         - nodejs14.x
-  #   license: MIT
+  - name: aws-lambda-layer
+    includeNames: /^sentry-node-serverless-\d+.\d+.\d+(-(beta|alpha)\.\d+)?\.zip$/
+    layerName: SentryNodeServerlessSDK
+    compatibleRuntimes:
+      - name: node
+        versions:
+          - nodejs10.x
+          - nodejs12.x
+          - nodejs14.x
```

Comment:
Lambda has `nodejs16.x` support now

Reply:
Yes, we know. We will address this in a future release.